x/auth/types/stdtx.go

```diff
@@ -59,16 +59,16 @@ func (tx StdTx) ValidateBasic() sdk.Error {
 	return nil
 }
 
-// countSubKeys counts the total number of keys for a multi-sig public key.
-func countSubKeys(pub crypto.PubKey) int {
+// CountSubKeys counts the total number of keys for a multi-sig public key.
+func CountSubKeys(pub crypto.PubKey) int {
```

Comment:
Why don't we move this to the `crypto` package and make it a function on `cypto.PubKey`?

Reply:
because it's not a property of a generalized pubkey and it wouldn't make sense to expand the pubkey interface type to account for this property of multisig pubkeys... could maybe be moved to crypto _as is_ though... but would rather not do that this PR

Reply:
lets move this to another PR...